tests/test_pvaccess.py

```diff
@@ -21,10 +23,19 @@ async def test_table_column_info(
         response_handler,
         command_queue,
         test_prefix,
-    ) = mocked_panda_standard_responses
+    ) = mocked_panda_multiple_seq_responses
+
     ctxt = Context("pva", nt=False)
 
-    table_value: Value = ctxt.get(test_prefix + ":SEQ:TABLE")
+    capturing_list: List = []
+    monitor = ctxt.monitor(test_prefix + ":SEQ1:TABLE", capturing_list.append)
+    start_time = datetime.now()
+    while not capturing_list and datetime.now() < start_time + timedelta(
+        seconds=TIMEOUT
+    ):
+        sleep(1)
+    monitor.close()
+    table_value = capturing_list[0]
```

Comment:
I think this would be much simpler if you used an `asyncio.Queue` instance and then use `await myQueue.get()`. You'll also need to wrap the whole thing in an `asyncio.wait_for` to add a timeout. But I think that's still simpler than implementing our own waiting logic.

Reply:
I thought we may want to use `p4p.client.thread` instead of asyncio, but happy to change.